mva/samples/sample.py

```diff
@@ -39,6 +40,8 @@
 from ..variables import get_binning, get_scale
 from mva.categories import get_trigger
 
+from moments import HCM
```

Comment:
This is still not included in your PR. I can't merge it until this is there or at least until you protect the code agains the absence of the module